tests/nightly/gpu/test_wizard.py

```diff
@@ -81,5 +86,49 @@ def test_retrieval(self):
         )
 
 
+@testing_utils.skipUnlessGPU
```

Comment:
why does this test need gpu?

Reply:
hmm good point, it doesn't. still pretty fast on cpu. i'll remove